packages/components/src/mobile/utils/alignments.native.js

```diff
@@ -13,6 +18,11 @@ export const WIDE_ALIGNMENTS = {
 		'core/button',
 	],
 	excludeBlocks: [ 'core/heading' ],
+	notInnerContainers: [
+		'core/separator',
+		'core/media-text',
+		'core/pullquote',
+	],
```

Comment:
Is the need for this arising as a result of not using the "allow-list"? It seems this is functioning in the `isContainerRelated` function as a kind of "deny-list". Won't this will incur the same maintenance cost described to be eliminated in the PR description? :thinking: 

I may be missing context, on why we need this, so maybe a comment could help. For example, in my understanding, media-text block _does_ use the `InnerBlocks` component, but it is listed here under `notInnerContainers` instead of `innerContainers`. Perhaps [the code comment on line 11](https://github.com/WordPress/gutenberg/pull/31625/files#diff-754c29386626931033a15770ba8a40952605ce00b9a96af11d474f18ed69d399R11) needs to be clarified? Since button block does not actually fit the criteria. Wdyt?

Reply:
I'm also a bit confused about the `notInnerContainers` and why only these three blocks are in the list

Reply:
@enejb Looks like `innerContainers` is not being used so I guess we can remove that?

Reply:
The `notInnerContainers` list is there so that the function works the same way as it did before.

All 3 blocks in `notInnerContainers` currently support full width alignment and the current function would return 'true' but the previous function would have returned 'false'. For this refactor I just wanted to remove the `innerContainers` dependency without changing the output of the function.

I would say that we can remove `notInnerContainers` but wasn't sure exactly what else could be affected here? 
Maybe we can test the 3 blocks and make sure that they still work as expected?

How can we do that?

Reply:
I think we should try to merge this PR and then look at doing a refactor of how margins are being calculated so that we can simplify things and add tests to make sure that things work as expected. For different cases. 

This PR will unblock the layout grid block work for now.

Reply:
Sounds good, I'll give it another quick look and ✅

Reply:
Thank you.